css/toggle-switch.css

```diff
@@ -23,6 +23,7 @@
 	.switch-light.switch-yoast-seo > span {
 		width: 250px;
 		border: 1px solid #ccc;
+		border-radius: 14px;
```

Comment:
Have you considered using 1em here instead of pixels?
The box has a min-height of 2em and a line-height of 2em, would feel right if this is in the same unit.